docs/ai/quickstarts/quickstart-ai-chat-with-data.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Quickstart - Build an Azure AI chat app with .NET about your data
+description: Create a simple chat app using your data and the .NET Azure OpenAI SDK.
+ms.date: 03/04/2024
+ms.topic: quickstart
+ms.custom: devx-track-dotnet, devx-track-dotnet-ai
+author: fboucher
+ms.author: frbouche
+# CustomerIntent: As a .NET developer new to Azure OpenAI, I want deploy and use sample code to interact to learn from the sample code.
+---
+
+# Build an Azure AI chat app with .NET about your data
```

Comment:
I'm not a fan of this title, the "about your data" feels added, but doesn't really flow with the rest of it. I suggest rephrasing this.

Reply:
I rephrase for: Get insight about your data from an .NET Azure AI chat app

an alternative could also be:  Build an .NET Azure AI chat app and get insight about your data